backend/app/api/ws/chat_namespace.py

```
@@ -797,35 +788,62 @@ async def on_chat_send(self, sid: str, data: dict) -> dict:
+                            team=team,
+                            user=user,
+                            message=payload.message,  # Original message
+                            payload=payload,
+                            task_room=task_room,
+                            namespace=self,
+                            user_subtask_id=user_subtask_id_for_context,  # Pass user subtask ID for unified context processing
+                            auth_token=auth_token,  # Pass original JWT token from WebSocket session
+                        )
+                    except Exception as e:
+                        logger.exception(
+                            f"[WS] chat:send AI trigger failed: task_id={task.id}, error={e}"
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)